knora/dsplib/utils/xml_upload.py

```diff
@@ -391,12 +374,15 @@ def _upload_resources(
             try:
                 img: Optional[dict[Any, Any]] = try_network_action(
                     action=lambda: sipi_server.upload_bitstream(filepath=os.path.join(imgdir, resource.bitstream.value)),
-                    failure_msg=f'ERROR while trying to create resource "{resource.label}" ({resource.id}).'
+                    failure_msg=f'ERROR while trying to upload file "{resource.bitstream.value}" of resource '
+                                f'"{resource.label}" ({resource.id}).'
                 )
             except BaseError as err:
                 print(err.message)
                 failed_uploads.append(resource.id)
                 continue
+            bitstream_size_uploaded_mb += round(next(bitstream_all_sizes_iterator), 1)
```

Comment:
I'm just curious: why do you need this information? Is it worth the time it takes to calculate this information? Have you measured it?

Reply:
cool, then it seems to be a nice feature for the user who uploads data